src/main/java/org/terasology/launcher/settings/BaseLauncherSettings.java

```diff
@@ -348,6 +377,12 @@ public synchronized void setKeepDownloadedFiles(boolean keepDownloadedFiles) {
         properties.setProperty(PROPERTY_SAVE_DOWNLOADED_FILES, Boolean.toString(keepDownloadedFiles));
     }
 
+    @Override
+    public synchronized void setShowPreReleases(boolean selected) {
+        showPreReleases.setValue(selected);
+        properties.setProperty(PROPERTY_SHOW_PRE_RELEASES, Boolean.toString(selected));
```

Comment:
This PR introduces the use of JavaFX Properties to the LauncherSettings interface, which seems like an appropriate usage.

The API question I have is: Why use a read-only property if you also expose a setter for it?

Is two-way data binding a concept that died with Angular.js?

(I'm bringing this up in the interest of interface design because you're doing a new type of thing on this class, but I won't make it a blocking concern for this release.)

Reply:
I'm not really sure about this either ... I'd like to have a "single source of truth" for the settings in code somewhere, and I want to avoid that arbitrary places just set/change the property (although they shouldn't). 

I'm not sure whether this is a good abstraction level (probably not), and I'd like to explore the design of `LauncherSettings` more. Can we use package visibility rules to expose a full property to some part of the code, but only read-only properties to others? Is that necessary at all (we can change the settings anyways via the setter).

Looking forward to more discussions on this :nerd_face: